src/SFA.DAS.Roatp.Functions/ApplicationExtract.cs

```diff
@@ -49,6 +52,7 @@ public ApplicationExtract(ILogger<ApplicationExtract> log, ApplyDataContext appl
 
                 await EnqueueApplyFilesForExtract(applyFileExtractQueue, answers);
                 await SaveExtractedAnswersForApplication(applicationId, answers);
+                await ExtractOrganisationManagementForApplication(applicationId, answers);
```

Comment:
Perhaps we need to refactor SaveChanges out here so everything goes in a single transaction.

Reply:
Done

Reply:
That not correctly done. Ping me when you start looking at this.

Reply:
Added transaction is added